### DISCUSSION TITLE
GPG Signature

### DISCUSSION THREAD

**Discussion by itinfosec** [standardnotes/forum#2773] [Suggestions]
Hey! Would it be possible to add GPG signatures to the file installers on GitHub please so that we can verify a download more securely.

**Comment by atmoio** [comment]
macOS and Windows builds are signed and the OS will let you know if there was tampering. Checksums are also available in the release assets under SHA256SUMS: https://github.com/standardnotes/desktop/releases

**Comment by itinfosec** [comment]
Sorry I don't think you understood what I meant. What I was asking for is for the releases to have a GPG signature file (.sig file) for verification using a public GPG key. This would make it more secure for verifying that a release has indeed come from yourselves and hasn't been compromised.
